action.yaml

```diff
@@ -5,6 +5,9 @@ inputs:
   test:
     description: 'name of the test to run (azure)'
     required: true
+  TEST_TYPE:
```

Comment:
why is this in caps?

Reply:
copy&paste error fixed with https://github.com/T-Systems-MMS/terratest-action/pull/10